workbench-cypress/cypress/e2e/awarmup/index.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { runTests } from "@utils/index";
+import {
+  shouldBeVisible,
+  shouldClickAndFind,
+  shouldFindAndContain,
+  shouldFindButton,
+  shouldFindCheckbox,
+  shouldTypeString,
+} from "@utils/sharedTests/sharedFunctionsAndVariables";
+import { Tests } from "@utils/types";
+
+// This should be the first test to set up the connection
```

Comment:
Hm using the file order to run these first when they're needed for the rest of the tests feels wrong. Seems like these should be explicitly run in some kind of setup function but not sure how to do that off the top of my head. Can do in a separate PR later if needed